### PR TITLE
add workspace managednetwork example yaml files

### PR DESCRIPTION
**Pull request by joshharrin**
# Description

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by kwucode — APPROVED** [review]

**Review by jloeffler7 — APPROVED** [review]

File: cli/resources/workspace/mvnet-allow-internet-outbound.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: myworkspace_dep
```
Comment (seanyao1) [inline cli/resources/workspace/mvnet-allow-internet-outbound.yml:1]: update the name.

File: cli/resources/workspace/mvnet-allow-internet-outbound.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+managed_network:
+  isolation_mode: allow_internet_outbound
+  outbound_rules:
+  - name: added-perule
+    destination:
+      service_resource_id: /subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/MyGroup/providers/Microsoft.Storage/storageAccounts/MyAccount1
+      spark_enabled: true
+      subresource_target: blob
+    type: private_endpoint
```
Comment (seanyao1) [inline cli/resources/workspace/mvnet-allow-internet-outbound.yml:10]: It's probably more readable to put type above destination

File: cli/resources/workspace/mvnet-allow-only-approved-outbound.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+name: myworkspace_dep
+managed_network:
+  isolation_mode: allow_only_approved_outbound
+  outbound_rules:
+  - name: added-servicetagrule
```
Comment (seanyao1) [inline cli/resources/workspace/mvnet-allow-only-approved-outbound.yml:5]: they can add outbound rules with IP prefixes too. Please add that into this example

Comment (joshharrin) [reply cli/resources/workspace/mvnet-allow-only-approved-outbound.yml:5]: no, this is not supported. we only support PE, ServiceTag, FQDN. not sure what you mean by this

Comment (joshharrin) [reply cli/resources/workspace/mvnet-allow-only-approved-outbound.yml:5]: ip address outbound rule does not seems supported from service, also do not think this is included in SDK/CLI either yet? will not include it in this example. let me know if I am missing something here

**Review by seanyao1 — APPROVED** [review]

**Review by fbsolo-ms1 — APPROVED** [review]

**Pull request merged**